.evergreen/config.yml

```diff
@@ -759,7 +764,8 @@ buildvariants:
 - matrix_name: "mongo-40-php7"
   matrix_spec: {"os-php7": "*", "versions": ["4.0", "4.2", "latest"], "php-versions": "7.3" }
   exclude_spec:
-     - {"os-php7": "rhel74-zseries", "versions": ["4.0", "4.2", "latest"], "php-versions": "7.3"}
+    - {"os-php7": "rhel74-zseries", "versions": ["4.0", "4.2", "latest"], "php-versions": "7.3"}
```

Comment:
Noted that the indentation is all that changed for this line. Two spaces seems correct.